08-projects.Rmd

```diff
@@ -4872,7 +4893,9 @@ The data in that folder has been added to the `elections` database in the `elect
 
 ##### 3. Write a SQL query that counts the number of donations (rows) that are from Indiana. How many donations are from Indiana? Rewrite the query and create an _alias_ for our field so it doesn't read `COUNT(*)` but rather `Indiana Donations`. 
 
-**Relevant topics:** [sql](#sql), [where](#sql-where), [aliasing](#sql-aliasing)
+**Hint:** You may enclose an alias name in quotation marks (single or double) when the name contains space.
```

Comment:
I added the hint simply from my experience: Previously I only have tried one-word alias names and thus not used quotation marks.